doc/nrf/app_dev/device_guides/fem/fem_nrf21540_gpio.rst

```diff
@@ -74,3 +74,44 @@ To use nRF21540 in GPIO mode, complete the following steps:
       };
 
    The pins defined in the GPIO forwarder node in the application core's devicetree file must match the pins defined in the FEM nodes in the network core's devicetree file.
+
+#. On nRF54L devices, the GPIO pins of the SoC selected to control ``tx-en-gpios``, ``rx-en-gpios`` and ``pdn-gpios`` must support GPIOTE.
+   For example, on the nRF54L15 device, only pins belonging to GPIO P1 or GPIO P0 can be used and GPIO P2 pins cannot be used due to lack of related GPIOTE peripheral.
+   It is recommended for mentioned purpose to use these GPIO pins that belong to the PERI Power Domain of the nRF54L device.
+   For example, on the nRF54L15, these are pins belonging to GPIO P1.
+   Using pins belonging to Low Power Domain (GPIO P0 on nRF54L15) is supported but requires more DPPI and PPIB channels of the SoC.
+   The nRF54L devices contain only four PPIB channels between PERI Power Domain and Low Power Domain.
+   Due to this limitation, only two out of three pins from group ``tx-en-gpios``, ``rx-en-gpios`` and ``pdn-gpios`` (for example, ``tx-en-gpios`` and ``rx-en-gpios``) can be controlled by GPIO P0.
```

Comment:
```suggestion
   Due to this limitation, only two out of three pins from groups ``tx-en-gpios``, ``rx-en-gpios`` and ``pdn-gpios`` (for example, ``tx-en-gpios`` and ``rx-en-gpios``) can be controlled by GPIO P0.
```

Reply:
I think the `group` should still be a singular form. The three names `tx-en-gpios` `rx-en-gpios` `pdn-gpios` make _the group_.  The "gpios" suggesting plural part of name is irrelevant here. For example, the `tx-en-gpios` is one member of _the group_.

Reply:
I stand corrected. My thinking was that each of those items was a group.